### PR TITLE
Dump docker image list

### PR DESCRIPTION
**Pull request by YangLu1031**
Need to dump docker image list for analysis:
https://github.com/kubernetes/kubernetes/issues/88011

### CODE REVIEW

File: cli_tools/diagnostics/main_windows.go [hunk 1]
```diff
@@ -256,6 +256,14 @@ func getPlainEventLogs(evts []winEvt, errs chan error) []string {
 	return runAll(commands, errs)
 }
 
+// getDockerImagesList put docker images list file in logFolder channel and errors in error channel.
+func getDockerImagesList(logs chan logFolder, errs chan error) {
+	var commands = []runner{
+		cmd{`C:\Program Files\Docker\docker.exe`, "image list", "docker_images.log", false},
```
Comment (pjh) [inline cli_tools/diagnostics/main_windows.go]: Remind me, what happens if we invoke this and docker isn't installed on the VM? Does this lead to any failures or error output for the user, or do we just not do anything?

**Comment by pjh** [comment]
LGTM

File: cli_tools/diagnostics/main_windows.go (hunk 1 above)
Comment (YangLu1031) [reply cli_tools/diagnostics/main_windows.go]: It won't break anything, we will only log a general error message in serial port

Comment (pjh) [reply cli_tools/diagnostics/main_windows.go]: Would it be possible to suppress that error message? (same for other already-existing checks for docker / kubernetes logs). If there's a reasonable way to do this then that would be nice, but not a deal-breaker.

Comment (YangLu1031) [reply cli_tools/diagnostics/main_windows.go]: Yeah, we suppress it like other logs and print one general message like `Errors occured while collecting and zipping some logs...` in serial port.

**Comment by pjh** [comment]
@ning-yang @adjackura PTAL

**Comment by YangLu1031** [comment]
@adjackura thanks for approval, but why it's still not able to merge?

**Pull request merged**